container-admin/Dockerfile_client

```diff
@@ -56,16 +54,18 @@ RUN echo "postgres:5432:*:*:postgres" > /home/ubuntu/.pgpass && chmod 600 /home/
 
 
 # for youxia and the consonance command line on the main box
-RUN wget --no-verbose https://github.com/Consonance/consonance/releases/download/${consonance_version}/consonance && mv consonance /home/ubuntu/bin && chmod a+x /home/ubuntu/bin/consonance
-RUN wget --no-verbose --no-check-certificate https://github.com/Consonance/consonance/releases/download/${consonance_version}/consonance-client-${consonance_version}.jar && mv consonance-client-*.jar /home/ubuntu/.consonance/self-installs/consonance-client-${consonance_version}.jar
-RUN mkdir /home/ubuntu/.dockstore && printf "server-url: https://dockstore.org:8443\ntoken: dummy-token" > /home/ubuntu/.dockstore/config
-RUN mkdir /home/ubuntu/.dockstore/self-installs && wget --no-verbose --no-check-certificate https://artifacts.oicr.on.ca/artifactory/collab-release/io/dockstore/dockstore-client/{{ CONSONANCE_DOCKSTORE_VERSION }}/dockstore-client-{{ CONSONANCE_DOCKSTORE_VERSION }}.jar && mv dockstore-client-*.jar /home/ubuntu/.dockstore/self-installs/dockstore-client-{{ CONSONANCE_DOCKSTORE_VERSION }}.jar
-RUN wget --no-verbose https://github.com/ga4gh/dockstore/releases/download/{{ CONSONANCE_DOCKSTORE_VERSION }}/dockstore && mv dockstore /home/ubuntu/bin && chmod a+x /home/ubuntu/bin/dockstore
+#RUN wget --no-verbose https://github.com/Consonance/consonance/releases/download/${consonance_version}/consonance && mv consonance /home/ubuntu/bin && chmod a+x /home/ubuntu/bin/consonance
+
+# Note: hard-coded patch to download executable
+RUN wget --no-verbose https://github.com/Consonance/consonance/releases/download/2.0.0-alpha.15/consonance && mv consonance /home/ubuntu/bin && chmod a+x /home/ubuntu/bin/consonance
```

Comment:
If a temporary patch is added, maybe indicate which expected version will render the patch obsolete to make it easier for future someone to maintain this.